octodns/provider/nsone.py

```diff
@@ -0,0 +1,189 @@
+#
+#
+#
+
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from logging import getLogger
+from nsone import NSONE
+from nsone.rest.errors import ResourceException
+
+from ..record import Record
+from .base import BaseProvider
+
+
+class NsOneProvider(BaseProvider):
+    '''
+    NsOne provider
+
+    nsone:
+        class: octodns.provider.nsone.NsOneProvider
+        api_key: env/NS_ONE_API_KEY
+    '''
+    SUPPORTS_GEO = False
```

Comment:
it seems like it may be fairly easy to get geo support in, let us know if we can help with that!